picr22-enne-1-voistlust/motion.py

```diff
@@ -0,0 +1,127 @@
+import turtle
+import math
+import numpy as np
+import time
+import tkinter as tk
+import struct 
+import serial
+
+class IRobotMotion:
```

Comment:
Are you using this file? if not remove it.

Reply:
didnt use any files in picr22-enne-1-voistlust so removed it